inst/sql/impact_method2_metadata/coverage_pop.sql

```diff
@@ -0,0 +1,33 @@
+SELECT tab1.*,
+tab2.value AS population 
+FROM
+(SELECT DISTINCT vaccine, activity_type, 
+coverage.country, year, gavi_support, 
+gender.name as gender, (age_from + i - 1) as age, target, coverage
+FROM coverage_set 
+JOIN coverage 
+ON coverage.coverage_set = coverage_set.id 
+JOIN gender 
+ON gender.id = coverage.gender
+JOIN num
+ON num.i <= (age_to - age_from + 1)
+WHERE touchstone = $1
+AND vaccine NOT IN ('none', 'PCV', 'DTP3') 
+AND year BETWEEN $2 AND $3) As tab1
```

Comment:
At the top of the file, in sql comments, please add a list of numbered arguments, eg

```sql
-- Parameters:
-- 1: the touchstone id (including version, e.g., gavi...-1)`
-- 2: etc
```

Reply:
Done.